README.md

```diff
@@ -121,19 +108,66 @@ and the applications in those remote git repositories will be deployed.
 By default, [github.com/opendatahub-io/ai-edge](https://github.com/opendatahub-io/ai-edge) is configured;
 edit the URLs to match your repositories.
 
-Then run as a regular (non-admin) user
+Then, as an admin user, run:
```

Comment:
This is an interesting part: it will at least have to be a user with permissions on ApplicationSets in the `openshift-gitops` namespace. This may have UX/UI implications when we go to integrate it more tightly with Open Data Hub.

Reply:
Which level of Admin user is required, cluster or project?

Reply:
I've updated it to try to make it clearer that what's needed is a user with permissions on certain resources in the openshift-gitops namespace. Please let me know if the new change doesn't clarify things for you, thanks :)